docs/introduction/index.md

```diff
@@ -0,0 +1,130 @@
+
+* Stability and consistency: .NET provides standardized programming frameworks across platforms that are stable, feature-rich, and easy to use.
+* Modern innovative languages: .NET languages are constantly improving with innovative new language features.
+* Industry-leading tools: The Visual Studio product family provides a fantastic .NET development experience across platforms on Windows, Linux, and macOS.
+* Speed and scalability: .NET has a strong history of performance, reliability, and security for app development. Using .NET as a full-stack solution makes it easier to build fast, reliable, and secure apps.
+* Full-stack development that leverages existing skills: C#/Razor developers use their existing C#/Razor skills to write client-side code and share server and client-side logic among apps.
+* Wide browser support: Blazor runs on .NET using open web standards in the browser with no plugins and no code transpilation. It works in all modern web browsers, including mobile browsers.
+
+## Running .NET in the browser
+
+Running .NET code inside web browsers is made possible by a relatively new technology, WebAssembly (abbreviated *wasm*). WebAssembly is an open web standard and is supported in web browsers without plugins. WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed.
+
+Security isn't a major concern because WebAssembly isn't ordinary assembly code (for example, x86/x64)&mdash;WebAssembly is a new bytecode format that accesses browser functionality with the same capabilities as JavaScript.
+
+When a Blazor app is built and run in a browser:
+
+1. C# code files and Razor files are compiled into .NET assemblies.
+1. The assemblies and the .NET runtime are downloaded to the browser.
+1. Blazor uses JavaScript to bootstrap the .NET runtime and configures the runtime to load required assembly references. Document object model (DOM) manipulation and browser API calls are handled by the Blazor runtime via JavaScript interoperability.
+
+## Browsers that don't support WebAssembly
+
+The .NET runtime is supplied as a WebAssembly binary and an [asm.js](https://wikipedia.org/wiki/Asm.js)-based implementation. *asm.js* is a subset of JavaScript and can be executed by JavaScript runtimes in browsers going back several years. When Blazor loads in the browser, it checks for WebAssembly support. If WebAssembly isn't supported, the *asm.js* runtime is loaded. *asm.js* isn't always used because it's larger and slower than the WebAssembly runtime.
+
+## Blazor components
+
+In client-side web UI frameworks, apps are built with *components*. A component usually represents a piece of UI, such as a page, dialog, or data entry form. Components can be nested, reused, and shared between projects.
```

Comment:
Ok ... I wasn't sure if that was *always* true. I mean one could have a Razor file included without markup, and I wasn't aware that it might not be called a "component" if that were the case. Therefore, I thought Steve said "usually" ... but I may have added that. I'll use your language there.